src/main/scala/com/gu/rcspollerlambda/Lambda.scala

```diff
@@ -28,16 +28,15 @@ object Lambda extends Logging with Config {
         // TODO: Alert on error
         logger.error(err.message)
       },
-      lastId => {
-        // TODO: Save lastId to db (once endpoint is ready)
-        logger.info(s"Lambda run successfully.")
+      {
+        case Some(id) =>
+          DynamoDB.saveLastId(id)
+          logger.info(s"Lambda run successfully.")
+        case None =>
+          // TODO: Alert on error
```

Comment:
For future reference, we'll definitely want to identify an alarm strategy for when this breaks to alert the team.